src/main/scala/uk/gov/homeoffice/drt/utils/AirportUtil.scala

```diff
@@ -0,0 +1,30 @@
+package uk.gov.homeoffice.drt.utils
+
+import uk.gov.homeoffice.drt.model.{Airport, Port}
+
+import scala.io.Source
+
+object AirportUtil {
+
+  lazy val source = Source.fromResource("airports.dat")
+
+  lazy val airportDetails: Seq[Airport] = source.getLines().map { line =>
+    val a: Array[String] = line.split(",").map(_.replace("\"", "").trim)
+    Airport(a(1), a(2), a(3), a(4), a(5), a(11))
+  }.toList
+
+
+  def getPortListForCountry(implicit country: String) = {
+    airportDetails.filter(_.country == country).filterNot(a => a.iata.isEmpty && a.icao.contains("\\N")).map { a =>
```

Comment:
this looks like it can't ever be true:
`a.iata.isEmpty && a.icao.contains("\\N")`

ie it can't be both empty and containing some characters

Reply:
maybe there's a test missing for what this is trying to do

Reply:
As we are getting port detail from aiport.dat and there row where iata is empty but icao is \\N . I have added test of Maldives which this rows.